### PR TITLE
[Manual Searc] Split ManualSearch and ForcedSearch threads

### PR DESCRIPTION
**Pull request by fernandog**
Fixed forced_search counting queue with manual search queue. and manualsnatch couting as manualsearch

with this:
- forced_search queue is only for forced searches
- manual_search queue is only for manual searches
- manual snatch doesn't count as search queue

### CODE REVIEW

**Comment by p0psicles** [comment]
changing https://github.com/pymedusa/SickRage/blob/more_rename/sickbeard/search_queue.py#L180 to:
self.name = 'FORCEDSEARCH-' + str(show.indexerid) if not manual_search else 'MANUALSEARCH-' + str(show.indexerid), would rename the queueItem. That's probably enough to show it as a manualSearch in logs.

File: sickbeard/manual_search.py [hunk 1]
```diff
@@ -136,11 +137,12 @@ def collectEpisodesFromSearchThread(show):
 
     # Finished Searches
     searchstatus = SEARCH_STATUS_FINISHED
-    for search_thread in sickbeard.search_queue.FORCED_SEARCH_HISTORY:
+    for search_thread in sickbeard.search_queue.MANUAL_SEARCH_HISTORY:
         if show and not str(search_thread.show.indexerid) == show:
             continue
 
-        if isinstance(search_thread, sickbeard.search_queue.ForcedSearchQueueItem):
+        if isinstance(search_thread, sickbeard.search_queue.ManualSearchQueueItem) or
```
Comment (p0psicles) [inline sickbeard/manual_search.py]: can be passed as a tuple: if isinstance(search_thread, (sickbeard.search_queue.ManualSearchQueueItem, sickbeard.search_queue.ForcedSearchQueueItem))

File: sickbeard/search_queue.py [hunk 2]
```diff
@@ -110,7 +112,7 @@ def queue_length(self):
                 length['forced_search'] += 1
             elif isinstance(cur_item, FailedQueueItem):
                 length['failed'] += 1
-            elif isinstance(cur_item, ManualSearchQueueItem):
+            elif isinstance(cur_item, ManualSearchQueueItem) or isinstance(cur_item, ManualSnatchQueueItem):
```
Comment (p0psicles) [inline sickbeard/search_queue.py]: pass as tuple

**Comment by fernandog** [comment]
@p0psicles done and working. did the tuple as you suggested.

**Comment by p0psicles** [comment]
I'm not convinced you should create a separate type search queue item for something that's almost similar, just for convenience. If they are alike, we should treat them alike. This is just wasting code.

**Comment by medariox** [comment]
@p0psicles 
I agree, we should make a more generic class and extend it as needed.

**Comment by fernandog** [comment]
i don't agree they are alike. but if you all decide its not good close PR

**Pull request closed without merging**